Jenkinsfile_parameterized

```diff
@@ -16,4 +37,5 @@ properties([
 
 withParameterizedPipeline(params.TYPE, params.PRODUCT_NAME, params.APP, params.ENVIRONMENT, params.SUBSCRIPTION) {
   enableDbMigration()
+  loadVaultSecrets(secrets)
```

Comment:
you could add slack notification `enableSlackNotifications(..)`

Reply:
It is there in Jenkinsfile_CNP. Maybe it would be better not to have notifications for our manually triggered Sandbox builds?